crates/bevy_remote/src/builtin_methods.rs

```diff
@@ -282,35 +291,48 @@ pub fn process_remote_get_request(In(params): In<Option<Value>>, world: &World)
     let mut response = BrpGetResponse::default();
 
     for component_path in components {
-        let reflect_component = get_reflect_component(&type_registry, &component_path)
-            .map_err(BrpError::component_error)?;
+        match handle_get_component(component_path, entity, entity_ref, &type_registry) {
+            Ok(serialized_object) => response.extend(serialized_object.into_iter()),
+            Err(err) if strict => return Err(err),
```

Comment:
I understand not returning here if not strict, but doesn't it make sense to return the errors in the response even so? Maybe its unexpected to the user. There is the fact that before, we returned the error to the user (blocking the operation, which isn't ideal), and the user would have to solve this. But now, the user will receive some components, but others not. Wouldn't it make sense to return too the errors of this operation if not strict?

Reply:
That's a good point. In my setup I loop through the sent components and filter the ones received to find what ones failed but it would make more sense to include that in the response. I'll add that before merging.